client/serviceregistration/service_registration.go

```diff
@@ -71,34 +71,32 @@ func (a *AllocRegistration) Copy() *AllocRegistration {
 	return c
 }
 
-// NumServices returns the number of registered services.
+// NumServices returns the number of registered task AND group services.
+// Group services are prefixed with "group-".
```

Comment:
This is because group services are automatically copied into tasks?

Reply:
Yeah. I think there's room for some refactoring here to make things more clear, but that's a separate PR

Reply:
Makes sense. I wasn't looking for any changes here, just making sure I understood the code 👍